manifests/python.yml

```diff
@@ -8,67 +8,72 @@ tests/:
     api_security/:
       test_schemas.py:
         Test_Schema_Request_Body:
-          '*': missing_feature
+          "*": missing_feature
```

Comment:
Could you put back the single quotes? it make the review, and later git blame hard to read

Reply:
Done. But it would make sens to use automatic format/lint for the manifests at some point.